### DISCUSSION TITLE
create tooling to automatically compute project metrics on a monthly basis

### DISCUSSION THREAD

**Discussion by MrPowers** [mrpowers-io/community#9] [Ideas]
Metrics to track:

* number of contributors
* number of stars
* number of downloads (for Rust and Python, hard to compute this for Maven projects)

Would be awesome to have a report to all projects, so we can easily see what needs to be done to grow projects every month.

**Comment by jeffbrennan** [comment]
made a repo to work on this here: https://github.com/jeffbrennan/ampere

happy to transfer this to the mrpowers-io organization whenever

feel free to put feature requests in the issues!

**Reply by MrPowers** [reply]
Sweet, love it, keep us posted on your progress @jeffbrennan!
